allennlp/training/trainer.py

```diff
@@ -482,8 +498,15 @@ def _train_epoch(self, epoch: int) -> Dict[str, float]:
         logger.info("Training")
         train_generator_tqdm = Tqdm.tqdm(train_generator,
                                          total=num_training_batches)
+        cumulative_batch_size = 0
         for batch in train_generator_tqdm:
             batches_this_epoch += 1
+            if self._log_batch_size_period:
```

Comment:
Could you move this below to where the rest of the logging happens, and also send the values to tensorboard:

https://github.com/allenai/allennlp/pull/2119/files#diff-043dcd121296c3cef3f3ff8c74127ff1R557

Reply:
Done